usernames_maintainers.txt

```diff
@@ -10,4 +10,5 @@ cbezault
 MahmoudGSaleh
 mnatsuhara
 StephanTLavavej
+strega-nil

```

Comment:
I'm trying to understand why this is being done?

Reply:
It's because you requested changes with that account, and the script needs to categorize everyone who has ever requested changes as either a maintainer (to affect the Wait lines) or a contributor.

Reply:
I could drop the Wait lines entirely, if we no longer think that this information is worth displaying.

Reply:
Maybe only the remaining shown by default one is worth keeping.

Reply:
Also mouthy produced PRs would possibly be curious

Reply:
Ah, fair enough; I don't have a problem with it, then, seems reasonable.

Reply:
We talked about this at the weekly maintainer meeting - I'll remove the Wait line machinery in the next Status Chart PR. (In the future I could add monthly created PRs or other metrics; right now I'm not sure how useful they'd be.)